yarn-project/acir-simulator/src/client/private_execution.test.ts

```diff
@@ -815,7 +819,7 @@ describe('Private Execution test suite', () => {
       );
 
       expect(result.enqueuedPublicFunctionCalls).toHaveLength(1);
-      expect(result.enqueuedPublicFunctionCalls[0]).toEqual(publicCallRequest);
+      expect(result.enqueuedPublicFunctionCalls[0].toBuffer()).toEqual(publicCallRequest.toBuffer());
```

Comment:
For some reason, the `toEqual` was failing with the objects directly, even though they serialise to the same values 🤷